test/common/index.js

```diff
@@ -749,6 +749,13 @@ exports.skipIfInspectorDisabled = function skipIfInspectorDisabled() {
   }
 };
 
+exports.skipIf32Bits = function skipIf32Bits() {
```

Comment:
👍 

The file is `test/common/README.md`, I'll describe `skipIfInspectorDisabled` too.